src/rest_service.py

```diff
@@ -247,7 +262,9 @@ def get_search_create_study(search_space_json, operation):
             hpo_service.instance.newExperiment(id_, experiment_name, total_trials, parallel_trials, direction,
                                                hpo_algo_impl, objective_function, tunables, value_type)
             logger.info("Starting Experiment: " + experiment_name)
-            hpo_service.instance.startExperiment(experiment_name)
+            response = hpo_service.instance.startExperiment(experiment_name)
+            if response:
+                return response
```

Comment:
Can we explicitly return an error code here. It looks like we have several `if`'s which just assumes a null return

Reply:
Removed redundant checks and added comment for clarity